File: 11 INFORMATIKA/SIR SEPRI/laporan segitiga/p.py
user_input_1 = int(input("Masukkan sisi pertama : "))
user_input_2 = int(input("Masukkan sisi kedua : "))
user_input_3 = int(input("Masukkan sisi ketiga : "))

if((user_input_1**2 + user_input_2**2 == user_input_3**2)):
    print("Segitiga siku-siku")
elif((user_input_1 == user_input_2) and (user_input_2 == user_input_3) and (user_input_3 == user_input_1)):
    print("Segitiga sama sisi")
elif(user_input_1 == user_input_3 and user_input_1 != user_input_2 and user_input_3!=user_input_2):
    print("Segitiga sama kaki")
else:
    print("Segitiga sembarang")

File: 11 INFORMATIKA/SIR SEPRI/laporan segitiga/p.cpp
#include<iostream>
using namespace std;
int main(){
    int sisi1,sisi2,sisi3;

    cin>>sisi1;
    cin>>sisi2;
    cin>>sisi3;

    if(sisi1 == sisi3 && sisi1 != sisi2 && sisi3 != sisi2){
        cout<<"Segitiga Sama kaki";
    }
    else if (sisi1 == sisi2 && sisi2 == sisi3 && sisi3 == sisi1){
    cout<<"Segitiga sama sisi";
    }

    else if (sisi1*sisi1 + sisi2*sisi2 == sisi3*sisi3){
    cout<<"Segitiga siku siku";
    }
    
    else{
        cout<<"Segitiga Sembarang";
    }
    return 0;
}

// Pengujian :
// Pada pengujian kali ini, ketika kita memasukkan angka 3,3,3 akan menghasilkan hasil berupa segitga sama sisi
// Pada pengujian kali ini, ketika kita memasukkan angka 3,2,3 akan menghasilkan hasil berupa segitga sama kaki
// Pada pengujian kali ini, ketika kita memasukkan angka 3,4,5 akan menghasilkan hasil berupa segitga siku siku
// Pada pengujian kali ini, ketika kita memasukkan angka 2,3,5 akan menghasilkan hasil berupa segitga sembarang
